src/ceph-volume/ceph_volume/devices/lvm/batch.py

```diff
@@ -260,17 +257,8 @@ def __init__(self, argv):
         for dev_list in ['', 'db_', 'wal_', 'journal_']:
             setattr(self, '{}usable'.format(dev_list), [])
 
-    def get_devices(self):
-        # remove devices with partitions
-        devices = [(device, details) for device, details in
-                       disk.get_devices().items() if details.get('partitions') == {}]
-        size_sort = lambda x: (x[0], x[1]['size'])
-        return device_formatter(sorted(devices, key=size_sort))
-
     def print_help(self):
-        return self._help.format(
-            detected_devices=self.get_devices(),
-        )
+        return self._help
```

Comment:
Let's get rid of `print_help()`.
Simply assign `description=self._help,` in the `ArgumentParser` constructor call..